docs/source/ml.rst

```diff
@@ -77,6 +77,34 @@ method. If the class you're using doesn't have a ``predict_proba`` method,
 you can add one by wrapping it in a :class:`~sklearn.calibration.CalibratedClassifierCV`.
 
 
+Custom Dependencies
+-------------------
+
+Installing packages from PyPI is straightforward. You can specify a `dependencies`
+argument to `~civis.ml.ModelPipeline` which will install the dependencies in your runtime
+environment. VCS support is also enabled (see [docs](https://pip.pypa.io/en/stable/reference/pip_install/#vcs-support). Installing a remote git repository from, say, Github
+only requires passing the HTTPS URL in the form of, for example,
+`git+https://github.com/scikit-learn/scikit-learn`.
+
+CivisML will run `pip install [your package here]`. We strongly encourage you to pin
+package versions for consistency. Example code looks like:::
+
+  from civis.ml import ModelPipeline
+  from pyearth import Earth
+  deps = ['git+https://github.com/scikit-learn-contrib/py-earth.git']
```

Comment:
Hah. And this repo has no releases yet. We could still pull from a commit. How about `deps = ['git+https://github.com/scikit-learn-contrib/py-earth.git@da856e1']`? We should demonstrate best practices in the documentation.